src/auth_token.c

```diff
@@ -80,9 +80,9 @@ int requestNewAuthToken(char *newToken, size_t len, int r_count)
 		curl_easy_setopt(curl, CURLOPT_URL, get_parodus_cfg()->token_server_url);
 		curl_easy_setopt(curl, CURLOPT_TIMEOUT, CURL_TIMEOUT_SEC);
 
-		if(get_parodus_cfg()->webpa_interface_used !=NULL && strlen(get_parodus_cfg()->webpa_interface_used) >0)
+		if(getWebpaInterface() !=NULL && strlen(getWebpaInterface()) >0)
```

Comment:
Here , 3 times getWebpaInterface() is called with mutex lock unlock . can we use one call and assign value to one variable and use as it is same func .